packages/terra-framework-docs/src/terra-dev-site/doc/slide-panel-manager/SlidePanelManager.1.doc.mdx

```diff
@@ -2,9 +2,9 @@ import { Badge } from 'terra-slide-panel-manager/package.json?dev-site-package';
 
 import SlidePanelManagerPropsTable from 'terra-slide-panel-manager/src/SlidePanelManager?dev-site-props-table';
 
-import SlidePanelManagerExample from '../example/SlidePanelManagerExample?dev-site-example';
-import SlidePanelManagerSquishExample from '../example/SlidePanelManagerSquishExample?dev-site-example';
-import SlidePanelManagerAccessoryExample from '../example/SlidePanelManagerAccessoryExample?dev-site-example';
+import SlidePanelManagerExample from './example/SlidePanelManagerExample?dev-site-example';
+import SlidePanelManagerSquishExample from './example/SlidePanelManagerSquishExample?dev-site-example';
+import SlidePanelManagerAccessoryExample from './example/SlidePanelManagerAccessoryExample?dev-site-example';
 
```

Comment:
As we migrate documentations to terra-framework-docs (and terra-core-docs which many packages have already migrated) many of the links in the doc that reference other packages are broken (or will be broken once those package docs are migrated) because they still reference the doc in their own package instead of terra-framework-docs.

For example the reference link below for [DisclosureManager API documentation](https://engineering.cerner.com/terra-ui/#/components/terra-disclosure-manager/disclosure-manager/disclosure-manager-api) is currently working but will be broken once terra-disclosure-manager is migrated but by that time, this change is already merged.

One recent issue we had to fix was https://github.com/cerner/terra-core/pull/3594

 Is there a plan to go back to clean up all these broken links?

Reply:
Should we create a task for fixing broken links? In that way we can fix all broken links for all mono repos in one single PR. What do you think?